src/java.security.jgss/share/classes/sun/security/krb5/internal/crypto/dk/ArcFourCrypto.java

```diff
@@ -159,10 +158,7 @@ public int getChecksumLength() {
            System.arraycopy(ss, 0, new_ss, 0, ss.length);
            Ksign = getHmac(baseKey, new_ss);
         } catch (Exception e) {
-            GeneralSecurityException gse =
-                new GeneralSecurityException("Calculate Checkum Failed!");
-            gse.initCause(e);
-            throw gse;
+            throw new GeneralSecurityException("Calculate Checkum Failed!", e);
```

Comment:
Can you also fix the typo, "Checkum" should be "Checksum". Same comment applies to line 172.

Reply:
Sure. Fixed